assets/js/googlesitekit/components-gm2/Button/MaybeTooltip.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * MaybeTooltip component.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import Tooltip from '../Tooltip';
+
+const MaybeTooltip = ( {
```

Comment:
Our convention is to define components using the `function` keyword. Let's do that here.

Example:

```js
export default function MaybeTooltip
```